test/h1/config.xml

```diff
@@ -37,8 +37,6 @@
     </device>
     <gui enabled="true" tls="false">
         <address>127.0.0.1:8081</address>
-        <user>testuser</user>
-        <password>$2a$10$7tKL5uvLDGn5s2VLPM2yWOK/II45az0mTel8hxAUJDRQN1Tk2QYwu</password>
```

Comment:
Nopes. The HTTP tests use this to test that authentication in fact works. I usually use `-home h2` when I do manual testing and don't want to bother with the auth; otherwise this is testuser / testpass.